tests/paste_events/_helpers/pasting.js

```diff
@@ -0,0 +1,72 @@
+/* exported assertPasteEvent, pasteFiles */
+
+'use strict';
+
+function assertPasteEvent( editor, eventData, expected, message, async ) {
```

Comment:
Hmm.  ok, don't bother with the spacing stuff, but maybe just add a comment with the original source path.